packages/vm/src/testing/storage.rs

```diff
@@ -21,8 +23,8 @@ impl MockStorage {
 }
 
 impl ReadonlyStorage for MockStorage {
-    fn get(&self, key: &[u8]) -> FfiResult<Option<Vec<u8>>> {
-        Ok(self.data.get(key).cloned())
+    fn get(&self, key: &[u8]) -> FfiResult<(Option<Vec<u8>>, u64)> {
+        Ok((self.data.get(key).cloned(), key.len() as u64))
```

Comment:
Can you pull put `let gas_cost = key.len() as u64;` to make clear why the length is used there?

Reply:
fixed in 8c571ed